src/blinx.js

```diff
@@ -158,6 +157,12 @@ let _callRender = function (module, placeholderResponse) {
 		}
 
 		res();
+
+		module.lifeCycleFlags.rendered = true;
+		_emitLifeCycleEvent(module, "_READY");
+
```

Comment:
We dont require these 2 loc as its redundant. we are already changing state above.